### DISCUSSION TITLE
Can AllenNLP run on MacBook Air without GPU?

### DISCUSSION THREAD

**Discussion by SimLif** [allenai/allennlp#5619] [Q&A]
I try to run `quick_start/train.py` in macbook air without gpu, but an ImportError appeared as follows：
```shell
ImportError: cannot import name 'ProcessGroup' from 'torch.distributed'
```
So whether Allanlp can only run on a computer with GPUs?

**Comment by epwalsh** [comment]
This is an issue with `torch.distributed`, and duplicated of https://github.com/allenai/allennlp/issues/5602

**Reply by SimLif** [reply]
Thank you, I got it.
